emergence_lib/src/graphics/ui/main_overlay.rs

```diff
@@ -0,0 +1,127 @@
+//! Main in-game player GUi
+
+use bevy::prelude::*;
+
+/// Create states to allow for switching GUI overlay
+pub enum OverlayState {
+    /// Overlay shown over the main game window
+    MainOverlay,
+
+    /// Shows what is releasing an attract intent
+    PheromoneOverlay1,
```

Comment:
What if?

```suggestion
    PheromoneAttract,
```

because `Overlay` is already in the enum name?

Similarly for other enum variants here

Reply:
Agreed, and remember that we swapped to Intent :)

Reply:
I went ahead and did so